ui/app/styles/components/ui-wizard.scss

```diff
@@ -173,3 +173,67 @@
 .wizard-instructions {
   margin: $size-4 0;
 }
+
+.selection-summary {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.time-estimate {
+  display: flex;
+  align-items: center;
+  color: $grey-light;
+  font-size: 12px;
```

Comment:
We try to do most of our sizing for fonts and spacing in `rems` or even with the `$size-N` vars from bulma_variables file if those fit ... that said I forget what size corresponds to 12px 😂 - I think it's maybe `$size-8` ?